ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/question_answering/QuestionAnsweringModel.java

```diff
@@ -47,50 +61,220 @@ public void warmUp(Predictor predictor, String modelId, MLModelConfig modelConfi
             throw new IllegalArgumentException("model id is null");
         }
 
+        // Initialize model type from config if not set
+        if (modelConfig != null) {
+            this.modelConfig = modelConfig;
+        }
+
         // Create input for the predictor
         Input input = new Input();
         input.add(DEFAULT_WARMUP_QUESTION);
         input.add(DEFAULT_WARMUP_CONTEXT);
+        input.add("0");
```

Comment:
can we add a comment why are we adding 0 here? and also put this as a static variable?

Reply:
Added into constant value with comment